tests/static/html/fluid.html

```diff
@@ -0,0 +1,32 @@
+<html>
```

Comment:
I don't see that this fixture is used in a test - how were you planning to inspect the component?